templates/security/fedramp.html

```diff
@@ -0,0 +1,296 @@
+{% extends "security/base_security.html" %}
+
+{% block title %}Enable FedRAMP with Ubuntu Pro | Security{% endblock %}
+
+{% block meta_description %}
+  Meet key FedRAMP security baseline controls with Ubuntu Pro: vulnerability patching, FIPS certified crypto modules, and system hardening.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1eJxGEthPLxG2CjFXTXLk6QF97DNh4VyZZ-AG2yIq8v4/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <div class="p-suru--25-75 p-section">
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h1>Enable FedRAMP compliance with Ubuntu Pro</h1>
+      </div>
+      <div class="col">
+        <p>
+          Achieve FedRAMP Authority to Operate and bring your cloud service offerings to the US Federal Government marketplace with the help of Ubuntu Pro. Meet the most demanding security baseline controls within your technology stack.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/support/contact-us?product=support-overview"
+             class="p-button--positive">Contact us</a>
+          <a href="/pro/subscribe" class="p-button">Get Ubuntu Pro</a>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row--25-75 p-section">
+      <hr />
+      <div class="col">
+        <h2 class="p-heading--5">
+          Why choose Canonical
+          <br />
+          for your FedRAMP journey
+        </h2>
+      </div>
+      <div class="col">
+        <p class="p-heading--2">Security patching made easy</p>
+        <hr class="is-muted" />
+        <p class="p-heading--2">FIPS-certified crypto modules</p>
+        <hr class="is-muted" />
+        <p class="p-heading--2">System hardening</p>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="u-fixed-width">
+      <h2 class="p-text--small-caps">Trusted by professionals</h2>
+    </div>
+    <div class="row--25-75">
+      <hr />
+      <div class="col">
+        <img src="https://assets.ubuntu.com/v1/021b5d90-acqua%20logo.png"
```

Comment:
Not sure if I'm understanding the doc correctly, but it looks like these might need to be wrapped in links